tests/collections/client.test.ts

```diff
@@ -54,9 +57,35 @@ describe('StargateMongoose - collections.Client', () => {
         done();
       });
     });
+    it('should initialize a Client connection with a uri using connect with overrides and a callback', done => {
+      const AUTH_TOKEN_TO_CHECK = "123";
+      const KEYSPACE_TO_CHECK = "keyspace1";
+      const BASE_API_PATH_TO_CHECK = "baseAPIPath1";
+      const LOG_LEVEL_TO_CHECK = "info";
+      const AUTH_HEADER_NAME_TO_CHECK = "x-token";
+      Client.connect(astraUri, {
+          applicationToken: AUTH_TOKEN_TO_CHECK,
+          keyspaceName: KEYSPACE_TO_CHECK,
+          baseApiPath: BASE_API_PATH_TO_CHECK,
+          logLevel: LOG_LEVEL_TO_CHECK,
+          authHeaderName: AUTH_HEADER_NAME_TO_CHECK
+        },
+        (err, client) => {
```

Comment:
Looks good. I'd just recommend avoiding using callbacks here. Mongoose is removing support for callbacks in the upcoming 7.0.0 release, so let's tweak this to use promises now so we don't have to later.

Reply:
@vkarpov15 Got it! I changed the callback based test to promise based one now.